src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +58,38 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
-        movieSessionService.add(tomorrowMovieSession);
+        final MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
+        final MovieSession movieSession = movieSessionService.add(tomorrowMovieSession);
```

Comment:
```suggestion
        movieSessionService.add(tomorrowMovieSession);
```
you can later use `tomorrowMovieSession`